src/components/shared/qrcode.tsx

```diff
@@ -32,8 +32,9 @@ export function QRCode({ text, options = {}, canvasProps }: QRProps) {
         if (!ref.current) return
 
         qr.toCanvas(ref.current, text, options, () => {
```

Comment:
https://www.npmjs.com/package/qrcode#tocanvascanvaselement-text-options-cberror

```suggestion
        qr.toCanvas(ref.current, text, options, (error) => {
            if (error) { .... }
```

Reply:
or https://www.npmjs.com/package/qrcode#todataurltext-options-cberror-url ?

Reply:
done